src/DynamoManipulation/TranslationGizmo.cs

```diff
@@ -51,6 +51,20 @@ public interface IGizmo
         /// <param name="factory">Render package factory</param>
         /// <returns>List of render packages.</returns>
         IEnumerable<IRenderPackage> GetDrawables(IRenderPackageFactory factory);
+
+        /// <summary>
+        /// Highlight gizmo drawables or create transient geometry to highlight the gizmo during mouse over
+        /// </summary>
+        /// <param name="backgroundPreviewViewModel"></param>
+        /// <param name="factory"></param>
+        /// <returns></returns>
+        void HighlightGizmo(IWatch3DViewModel backgroundPreviewViewModel, IRenderPackageFactory factory);
```

Comment:
Since this method is already on Gizmo class, we can name it as Highligh() and Unhilight()